multimedia/minisatip/Makefile

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (C) 2010-2012 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=minisatip
+PKG_VERSION:=0.5
+PKG_RELEASE:=1
+
+PKG_SOURCE_URL:=https://github.com/catalinii/minisatip/archive
+PKG_SOURCE:=$(PKG_VERSION).tar.gz
+PKG_HASH:=7cbad5397fd7ba1783a165b382e6735350011399b9c9aa6acf54592a4ea58e76
+
+PKG_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/minisatip
+  TITLE:=SATIP server
+  URL:=https://github.com/catalinii/minisatip
+  SECTION:=multimedia
+  CATEGORY:=Multimedia
+endef
+
+
+define Build/Configure
```

Comment:
Replace section with ```CONFIGURE_ARGS += --with-linux-headers=$(LINUX_DIR)```?

Reply:
You also probably need to review the other options so it doesn't end up pulling in libraries that aren't specified as dependencies.